Specification/8._Hardware_design_recommendations.md

```diff
@@ -4,22 +4,22 @@ permalink: /Specification/8._Hardware_design_recommendations/
 
 # Hardware design recommendations
 
-The points concerning hardware design practices are listed in this chapter.
-Following them is highly recommended, because that would ensure the highest level of inter-vendor compatibility
-and would allow the developers to avoid some of the common pitfalls.
+This chapter contains best-practice recommendations for hardware design.
+
+Following these guidelines will ensure the highest level of inter-vendor compatibility
+and allow the developers to avoid many common design pitfalls.
 
 ## Physical connector definition
 
+**Do not deviate from the recommendations provided in this section, as this may impact cross-vendor compatibility.**
```

Comment:
This "message" was further down in text, but not good in the "flow". It feels like we've just mentioned vendor compatibility, but I assume this section in particular requires that people don't use a different connector/deviate from spec?
Depending on your intention you could remove this line or change to something like "Only use the standard connectors defined in this section".

Reply:
Indeed, I think it should be safe to remove this line completely. The message is made sufficiently clear in the introduction.

Reply:
Great. I will remove.